Sources/SentrySwiftUI/SentryReplayView.swift

```diff
@@ -0,0 +1,43 @@
+#if SENTRY_NO_UIKIT
+import SentryWithoutUIKit
+#else
+import Sentry
+#endif
+import SwiftUI
+import UIKit
+
+@available(iOS 13, macOS 10.15, tvOS 13, *)
+struct SentryReplayView: UIViewRepresentable {
```

Comment:
`m`: Can we somehow add tests for that?

Reply:
It looks like it's possible. I will give it a try